### DISCUSSION TITLE
Submit app to GNOME Circle

### DISCUSSION THREAD

**Discussion by afranke** [johnfactotum/foliate#751] [General]
This application would be a good fit for [GNOME Circle](https://circle.gnome.org/), I suggest you apply.

**Comment by JohannPopper** [comment]
Foliate certainly deserves to be among showcased GNOME apps, and this could attract new helpers for future maintenance, if that is something the current owner wants. Foliate is a wonderful app that I want to use for years to come, so I would support every future-proofing action.

**Comment by julianfairfax** [comment]
Any update on this? It has been more than two years. The app is great and deserves a spot in Circle!

**Comment by johnfactotum** [comment]
Yes, I will probably submit it at some point, though I'd like to fix some of the accessibility issues first.

**Comment by julianfairfax** [comment]
In addition to joining Circle, it would also be nice if the app's code were moved to [GNOME World](https://gitlab.gnome.org/World). GitHub is not an open source platform, and being owned by Microsoft makes it more of a danger to open source projects, rather than a help.
